README.md

```diff
@@ -1,5 +1,68 @@
-### UFV CSA Website
+# UFV CSA Website
 
-This repository contains the source data for the UFV Computing Student Association's website.
+## Summary
 
-The website is currently in development. Pull requests for content, styling, or design changes are welcome.
+This repository contains the source data for the UFV Computing Student Association's website!
+
+### Goals
+
+This repository has a few key goals:
+
+1. Maintain the UFV CSA website with up-to-date and relevant content for CSA members.
+2. Provide CSA members with an opportunity to work with and learn valuable workplace skills including:
+   - Team Communication: How to effectively work with others on a real-world project.
+   - Git / Github Workflows: How to manage code-bases using git / github
+   - Realworld Web Technologies: How to work with modern web technologies like Gatsby.js, React, and Node.
+3. Add more goals / re-arrange
```

Comment:
@MJJBennett Was wondering if you had any goals in mind for this project or if you agreed with what I've listed here so far? Thanks!